### PR TITLE
[FIX] Unable to close opened chat by clicking on close button in Omnichannel

### LINKED ISSUE #21604: Close button at the header of chat in Omnichannel isn't working
### Description:

Can't close an opened chat by clicking on the close button.

### Steps to reproduce:

1. Go to Omnichannel --> Current chats
2. Click on any of the chats.
3. Click on Options --> Contact Chat History
4. Click on any of the chats from the list.
5. Try closing it by clicking on the close button.

### Expected behavior:

Clicking on the 'Close Button' should close the opened chat.  

### Actual behavior:

'Close Button' isn't responding.

https://user-images.githubusercontent.com/44610854/115016877-ab7e0800-9e83-11eb-8337-cf2916270773.mp4

### Server Setup Information:

- Version of Rocket.Chat Server: 3.14.0
- Operating System: Linux (Ubuntu 18.04)
- Number of Running Instances: 1
- NodeJS Version: 12.22.1
- MongoDB Version: 4.2.8

### Client Setup Information

- Browser Version: Google Chrome 90.0.4430.72
- Operating System: Linux (Ubuntu 18.04)

### PR DESCRIPTION
**Pull request by Divyani18**
## Proposed changes (including videos or screenshots)

Earlier, the Close button at the header of chat opened from Contact Chat History wasn't responding. Now, clicking on the Close Button closes the opened chat and takes back to the list of chats.

Before:

https://user-images.githubusercontent.com/44610854/115617286-3e250980-a2bf-11eb-98f3-7da6a42afbbb.mp4

After:

https://user-images.githubusercontent.com/44610854/115617360-5006ac80-a2bf-11eb-9c61-60b7fe22d227.mp4

## Issue(s)
Fixes #21604

### CODE REVIEW

**Comment by Divyani18** [comment]
@ggazzo Please review it and let me know if I need to make any changes.

File: package.json [hunk 1]
```diff
@@ -189,6 +189,8 @@
 		"codemirror": "^5.56.0",
 		"coffeescript": "^2.5.1",
 		"connect": "^3.7.0",
+		"cookie": "^0.4.1",
```
Comment (renatobecker) [inline package.json:192]: I don't think this dependency is part of the problem you're solving.

**Review by renatobecker — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (Divyani18) [reply package.json]: @renatobecker I have made the changes requested. 2 workflows require approval.

**Comment by MartinSchoeler** [comment]
Closing this since we refactored this view to use react, and the issue should be fixed with the refactor. Thanks for the contribution!

https://github.com/RocketChat/Rocket.Chat/pull/27302

**Pull request closed without merging**